descriptors/ModuleDescriptor-template.json

```diff
@@ -5,39 +5,60 @@
   "provides": [
     {
       "id": "calendar",
-      "version": "2.0",
+      "version": "3.0",
       "handlers": [
         {
           "methods": [
             "GET"
           ],
           "pathPattern": "/calendar/periods",
           "permissionsRequired": [
-            "calendar.collection.get"
+            "calendar.opening-hours.collection.get"
```

Comment:
Why is the permission for `/calendar/periods` naming `opening-hours`, yet `/calendar/periods/{servicePointId}/period` is named `periods`?

What type of record comes from each of these two endpoints?

Reply:
I'm not sure about the reasons why this naming was used for these endpoints, but actually `/calendar/periods` returns `opening-hour` records, 
when `/calendar/periods/{servicePointId}/period` returns `period` records.

Reply:
That is fair enough. I don't want to make this changer any bigger.